2024/FOSDEM/osf_on_amd_5th.md

```diff
@@ -0,0 +1,287 @@
+  (desktop & laptop), AMD Family 17h, Model 18h
+- **Cezanne** - Zen3 core architecture, Ryzen 5000 series (desktop & laptop),
+  AMD Family 19h, Model 50h
+- **Mendocino** - Zen2 core architecture, Ryzen 7000-series Mobile CPUs with
+  RDNA2 graphics, formerly **Sabrina**, AMD Family 17h, Model A0h
+- **Phoenix** - Zen4 core architecture, Ryzen 7000-series Mobile CPUs
+  with RDNA3 graphics, formerly **Morgana**, AMD Family 19h, Models 70h-7Fh?
+- **Glinda** - very new and very little information about it, also probably a
+  temporary codename
+
+---
+
+# AMD coreboot status - last year
+
+- family14, Trinity and Kabini removed from the master branch and moved to 4.18
+  branch
+- [boards
+  affected](https://doc.coreboot.org/releases/coreboot-4.19-relnotes.html#removed-mainboards):
+    * PC Engines apu1
+    * MSI MS-7721 (FM2-A75MA-E35)
+    * Lenovo AMD G505s
+    * HP Pavilion m6 1035dx
+    * ASUS F2A85-M (LE, PRO), A88XM-E and AM1I-A
+    * ASRock E350M1 and IMB-A180
+    * and others...
+- **Since then there were no AMD board removals yet**
+
+---
+
+# AMD coreboot status - last year
+
+- Starlabs could build coreboot firmware for their AMD laptops thanks to the
+  publication of Cezanne FSP to
+  [amd_blobs](https://github.com/coreboot/amd_blobs/tree/master/cezanne)
+  repository in September 2022
+    * [mb/starlabs/cezanne: Add Cezanne Byte Mk
+      I](https://review.coreboot.org/c/coreboot/+/69404/)
+    * [mb/starlabs/cezanne: Add Cezanne StarBook Mk VI
+      variant](https://review.coreboot.org/c/coreboot/+/68338/)
+    * **NO UPDATE TO THE PATCHES**
+- AMD Mendocino and Phoenix still in development with the former being in more
+  advanced state, FSP not published yet
+    * **FSP published for Mendocino, but not for Phoenix**
+- The FSP publication interval is quite long (1.5 a year between Picasso FSP
+  and Cezanne FSP release to public, and 1.25 a year after Cezanne APU release
+  FSP has been published)
+    * **Interval between Cezanne and Mendocino is only 5 months**
+    * **Mendocino is Zen2 while Cezanne is Zen3, so maybe not so big update**
+    * **Mendocino has been released to the market at the end of 2022**
+
+???
+
+Cezanne is newer core architecture, but Mendocino is a newer processor in
+terms of release date.
+
+---
+
+# AMD coreboot status - now
+
+- [patches covering KGPE-D16 bootblock support are out
+  there](https://review.coreboot.org/q/topic:kgpe-d16-bootblock)
+    * they have been abandoned because of lack of activity
+    * KGPE-D16 needs some love and attention, which, unfortunately, 3mdeb
+      can't humbly provide right now without any support from community
+- Marty Plummer ("hanetzer") is working on adapting AMD FSP for
+  Picasso/Cezanne to work on a non-Chromebook device ASRock x370 Killer SLI
+    * Join [Dasharo Matrix Space](https://matrix.to/#/#dasharo:matrix.org) or
+      [Dasharo vPubs](https://vpub.dasharo.com/o/1) to know more
```

Comment:
I guess this deserves more attention; I'm pretty sure there are some demo recordings from DUG#3

https://www.youtube.com/watch?v=LYPH2Oc6xqU&list=PLuISieMwVBpKevqcC9qEav9ZnCTwkZVW2&index=8

Reply:
There is also vPub 0x9 recording sent in newsletter

Reply:
I guess I haven't booked a ticket for DUG #4/vPub 0x9... I only have a ticker confirmation from DUG #3/vPub 0x8...

Reply:
I forwarded you an email.

Reply:
Thanks, added links: https://github.com/3mdeb/conferences/pull/20/commits/71382d62a8fcec201d0db24cc3ab111f4d499127#diff-a36529ceeba71601b1edbcceb8639bd8b743d579e8321eff59d469b53e6ad1d0R138-R139